package.json

```diff
@@ -77,6 +84,24 @@
     "husky": "^7.0.0",
     "lint-staged": "^11.2.3",
     "prettier": "^2.2.1",
-    "typescript": "^4.2.3"
+    "typescript": "^4.2.3",
+    "@testing-library/jest-dom": "^4.2.4",
+    "@testing-library/react": "^9.5.0",
+    "@testing-library/user-event": "^7.2.1",
+    "@types/jest": "^25.1.4",
+    "babel-eslint": "^10.0.3",
+    "cross-env": "^7.0.2",
+    "eslint-config-airbnb": "^18.2.1",
+    "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jsx-a11y": "^6.4.1",
+    "eslint-plugin-prettier": "^3.1.4",
+    "eslint-plugin-react-hooks": "^4.2.0",
+    "gh-pages": "^2.2.0",
+    "microbundle-crl": "^0.13.10",
+    "npm-run-all": "^4.1.5",
+    "react-scripts": "^3.4.1",
+    "@babel/core": "^7.11.1",
+    "@types/react-dom": "^16.9.7",
+    "babel-loader": "^8.1.0"
```

Comment:
I don't think these dependencies are required.

Reply:
tefi-app-0.1.1.tgz file removed
yarn lint command added again
packages/editor/dist directory is already removed.
empty interfaces are disabled off for the files
dependencies moved to dev dependencies
tslint.json deleted
PostCommentView.tsx file added again
unwanted dependencies removed